contracts/handler/TokenCallbackHandler.sol

```diff
@@ -43,6 +43,10 @@ contract TokenCallbackHandler is ERC1155TokenReceiver, ERC777TokensRecipient, ER
 
     /**
      * @notice Handles ERC777 Token callback.
+     * @dev Account that wishes to receive the tokens also needs to register the implementer (this contract) via the ERC-1820 interface registry.
+     *      From the standard: "This is done by calling the setInterfaceImplementer function on the ERC-1820 registry with the holder address as
+     *      the address, the keccak256 hash of ERC777TokensSender (0x29ddb589b1fb5fc7cf394961c1adf5f8c6454761adf795e67fe149f658abe895) as the
+     *      interface hash, and the address of the contract implementing the ERC777TokensSender as the implementer."
      * return nothing (not standardized)
```

Comment:
```suggestion
     * @return nothing (not standardized)
```

Reply:
My bad, maybe add it as a sentence, as this one seems like the end of the previous line.

Reply:
I removed it because it adds no value. We also don't specify empty returns in other contracts' NatSpec.